sdk/core/azure-core/src/main/java/com/azure/core/exception/AzureException.java

```diff
@@ -55,8 +55,12 @@ public AzureException(final String message, final Throwable cause) {
      * @param enableSuppression Whether suppression is enabled or disabled.
      * @param writableStackTrace Whether the exception stack trace will be filled in.
      */
-    public AzureException(final String message, final Throwable cause, final boolean enableSuppression,
-        final boolean writableStackTrace) {
+    public AzureException(
+        final String message,
+        final Throwable cause,
+        final boolean enableSuppression,
+        final boolean writableStackTrace
```

Comment:
I've never seen this style before. What I typically see is that the first parameter remains on line 58, and then all subsequent parameters align vertically with that, like this. The last parameter has the closing brace of the parameter list, and the opening bracket of the method definition, like this:

```suggestion
    public AzureException(final String message,
                          final Throwable cause,
                          final boolean enableSuppression,
                          final boolean writableStackTrace) {
```

Reply:
This should be doable, no strong opinions on design here